Enhanced and Refactored Pokemon Inventory Page

Markusch85 opened the pull request:
Closes issues
-------------
- Closes #1524 
- Closes #1604 

Changes
-------
- Added: Favorite Icon to Pokemon Inventory Page
- Added: Blue Background, if Pokémon was capture within the last 24 hours
- Added: new UserControl for Pivot Header
- Added: new Style for PivotHeaderItem
- Fixed: Hard coded Pokémon inventory capacity

Change Details
-------
I've also refactored a bit the Pokémon inventory page.
I've added two new Images: For Pokémon as favorite and for the Checkbox, if Pokémon is no favorite. So it Looks like the original one from Android or iOS.

Other informations
-------
Let's have a look, guys. ;)
![fixed](https://cloud.githubusercontent.com/assets/20780000/18419869/db6765ce-7864-11e6-9790-ecf14180e01e.png)

robipzo commented:
Is this circle also in the original app?sorry I'm not sure.But i like it :)

Markusch85 commented:
Thank you.
See #1524 The third Screenshot shows the circle.

Rover656 commented:
Could you try to get a gradient for the circle?

Markusch85 commented:
@Rover656 Sorry, I've tried to implement a glow effect, but I wasn't abled to do that. It'll be nice, if you have a tip for. I used LinearGradientBrush in XAML.

khunpanya commented:
@Markusch85 I don't know much about XAML. But maybe just a semi-transparent png would do?

robertmclaws commented:
The pivot header alignment is wrong in your screenshot. I don't think this PR will be accepted if it introduces new bugs. We've fixed that header alignment a number of times, I don't think anyone is going to want to fix it again.

ST-Apps commented:
Just adding that there's no radial gradient in UWP so you may want to create that blue fading image with Photoshop and use it to replace the circle.

Markusch85 commented:
@robertmclaws Sorry, my fault. I removed the style from custom.xaml. It is now fixed and I've updated the screenshot.

@ST-Apps I'm not very good with Photoshop or Gimp. Maybe somebody else is able to create such a background image.

ST-Apps commented:
This can work for now, I'll see if I can get that gradient before the release. Thanks.

ArthurSDev commented:
Add one thing like this to show when pokemon is on Gym ![sem titulo](https://cloud.githubusercontent.com/assets/18059954/18533282/2c4a3cdc-7ab8-11e6-8ce7-a656b01fbba4.png)
